recipes-kernel/linux/files/patches-5.10/0001-dmaengine-ti-k3-udma-glue-Add-function-to-get-device.patch

```diff
@@ -1,4 +1,4 @@
-From 3f592d3a8794d4db3192b71a7f8df363cb1c2c62 Mon Sep 17 00:00:00 2001
+From 274d819974b7cb1cfadf29a6312d0ede1119a8af Mon Sep 17 00:00:00 2001
```

Comment:
Per my experience, this change could be safely ignored.

BTW, @jan-kiszka , I think it might be a good idea to apply the --zero-commit option, see https://github.com/git/git/blame/master/Documentation/git-format-patch.txt#LL384C16-L384C16. What do you think?

Reply:
Good suggestion, indeed.

Reply:
Yes, good idea.